retail/retail-clickstream-application/datalayer/view_item.json

```diff
@@ -0,0 +1,22 @@
+{
+  "event_datetime":"2020-11-16 22:59:59", 
+  "event": "view_item",
+  "user_id": "UID00003",
+  "ecommerce": {
+    "items": [{
+      "item_name": "Donut Friday Scented T-Shirt", 
+      "item_id": "67890",
+      "price": 33.75,
+      "item_brand": "Google",
+      "item_category": "Apparel",
```

Comment:
We have item_category and then item_category_2 , could we have all as item_category_01 .. 05.